tests/Stubs/LoggerStub.php

```diff
@@ -0,0 +1,81 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\LoyaltyCorp\Search\Stubs;
+
+use EoneoPay\Externals\Logger\Interfaces\LoggerInterface;
+use Throwable;
+
+final class LoggerStub implements LoggerInterface
```

Comment:
Instead of adding this stub here, can you please send a PR to eonx-com/test-utils adding a general purpose logger that can be used everywhere?

There are already some example stubs in that package.

Reply:
Created https://github.com/eonx-com/test-utils/pull/14 to have the Logger stub in test-utils

Reply:
Tagged v0.2.6 with that pr merged

Reply:
Alright Thanks :D